pkg/project/runtime/runtime.go

```diff
@@ -47,6 +48,31 @@ const (
 
 var commonIgnore = []string{".nitric/", "!.nitric/*.yaml", ".git/", ".idea/", ".vscode/", ".github/", "*.dockerfile", "*.dockerignore"}
 
+func getDockerIgnores(dockerIgnorePath string, fs afero.Fs) ([]string, error) {
+	// Check if the file exists
+	exists, err := afero.Exists(fs, dockerIgnorePath)
+	if err != nil {
+		return nil, err
+	}
+
+	if exists {
+		// Read the file
+		content, err := afero.ReadFile(fs, dockerIgnorePath)
+		if err != nil {
+			return nil, err
+		}
+
+		// Split the content into lines
+		lines := lo.Filter[string](strings.Split(string(content), "\n"), func(line string, index int) bool {
+			return strings.TrimSpace(line) != ""
+		})
+
+		return lines, nil
+	}
+
+	return nil, nil
```

Comment:
return default value over nil, or an error if not exists and handle outside? double nil returns can be confusing and error prone (requires reading the code to understand the case, when you encounter a nil pointer exception from misuse).

Reply:
changed to use default